stable/traefik/README.md

```diff
@@ -70,7 +70,7 @@ The following tables lists the configurable parameters of the Traefik chart and
 | ------------------------------- | -------------------------------------------------------------------- | ----------------------------------------- |
 | `imageTag`                      | The version of the official Traefik image to use                     | `v1.1.2`                                  |
 | `serviceType`                   | A valid Kubernetes service type                                      | `LoadBalancer`                            |
-| `replicas`                      | The number of replicas of Traefik to run                             | `1`                                       |
+| `replicas`                      | The number of replicas of Traefik to run, __NOTE:__ full Traefik clustering not yet supported with can affect any configured Let's Encrypt setup, see Clustering section | `1` |
```

Comment:
Typo here. s/with/which/. If additionally you don't mind some formatting suggestions:

```
The number of replicas to run; __NOTE:__ Full Traefik clustering with leader election is not yet supported, which can affect any configured Let's Encrypt setup; see Clustering section
```